src/backend/distributed/sql/udfs/citus_stats_tenants/latest.sql

```diff
@@ -0,0 +1,66 @@
+-- cts in the query is an abbreviation for citus_stats_tenants
+CREATE OR REPLACE FUNCTION pg_catalog.citus_stats_tenants (
```

Comment:
could we use citus_stat_tenants (without the extra s) to be consistent with existing pg_stat_ and citus_stat_ views?

@mulander this seems like a typo?

Reply:
Merged a PR to fix this